test/simulator/timestamp/src/include/timestamp_manager.h

```diff
@@ -38,7 +39,8 @@ class timestamp_manager {
     /* Methods */
     public:
     static timestamp_manager &get_timestamp_manager();
-    void parse_config(const std::string &, std::map<std::string, std::string> &);
+    int parse_config(const std::string &, std::map<std::string, std::string> &,
```

Comment:
The output usually comes at the end, the `map` arg should be the last.

Reply:
Updated in [ce5139b](https://github.com/wiredtiger/wiredtiger/pull/8372/commits/ce5139be8bb0cbda4016b297614a1ca5a0c72f1e)